### PR TITLE
Enhancement: Enable and configure phpdoc_types_order fixer

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] enables and configures the `phpdoc_types_order` fixer
* [x] runs `make cs`

Follows #157.

💁‍♂️ For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.17.2/doc/rules/phpdoc/phpdoc_types_order.rst.

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -112,6 +112,10 @@ return $config
         'phpdoc_single_line_var_spacing' => true,
         'phpdoc_trim' => true,
         'phpdoc_types' => true,
+        'phpdoc_types_order' => [
+            'null_adjustment' => 'always_last',
```
Comment (localheinz) [inline .php_cs.dist:116]: This configuration results in the least number of changes.

**Review by pimjansen — APPROVED** [review]

**Comment by pimjansen** [comment]
Failing build

**Comment by localheinz** [comment]
@pimjansen 

I have regenerated the baseline following #184.

**Comment by localheinz** [comment]
Thank you, @pimjansen!

**Pull request merged**